packages/vue-vuetify/dev/renderers/TemperatureRenderer.vue

```diff
@@ -0,0 +1,77 @@
+<template>
+  <integer-control-renderer v-bind="$props">
+    <template #prepend>
+      <v-tooltip :text="tempTooltip" location="bottom">
+        <template #activator="{ props: tooltipProps }">
+          <v-icon
+            v-bind="tooltipProps"
+            :color="tempColor"
+            class="mr-1"
+            aria-hidden="true"
+            tabindex="-1"
+          >
+            {{ tempIcon }}
+          </v-icon>
+        </template>
+      </v-tooltip>
+    </template>
+
+    <template #append>
+      <v-icon>mdi-temperature-celsius</v-icon>
+    </template>
+  </integer-control-renderer>
+</template>
+
+<script setup lang="ts">
+import { rendererProps, useJsonFormsControl } from '@jsonforms/vue';
+import type { ControlElement } from '@jsonforms/core';
+import IntegerControlRenderer from '../../src/controls/IntegerControlRenderer.vue';
+import { VIcon, VTooltip } from 'vuetify/components';
+import { computed } from 'vue';
+
+const props = defineProps(rendererProps<ControlElement>());
+const { control } = useJsonFormsControl(props);
+
+// Determine temperature category (Celsius)
+const category = computed(() => {
+  const temp = control.value.data || 0;
+  if (temp <= 4) return 'freezing';
```

Comment:
Minor: Should match the labels below
```suggestion
  if (temp <= 5) return 'freezing';
```